### PR TITLE
Change CRDB to Active-Active; Replica Of to Active-Passive

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rs/administering/active-active.md [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 weight: $weight
 alwaysopen: false
 categories: ["RS"]
+aliases: /rs/administering/intercluster-replication/crdbs/
 ---
-Conflict Free Replicated Databases (CRDBs) provide a simple and
-effective way to replicate your data between one or more Redis
-Enterprise Software (RS) clusters.
+In Redis Enterprise, active-active geo-distribution is based on CRDT technology.
+The Redis Enterprise implementation of CRDT is called Conflict-Free Replicated Database (CRDB).
```
[the remark this answers is not in the document]
Comment (bmansheim) [reply content/rs/administering/active-active.md]: This is what Yiftach asked for as I recall.

File: content/rs/administering/active-active.md [hunk 2]
```diff
@@ -1,19 +1,17 @@
 ---
-Title: Administering CRDBs
+Title: Active-Active Geo-Distributed Redis (CRDB)
```
[the remark this answers is not in the document]
Comment (bmansheim) [reply content/rs/administering/active-active.md]: I'll ask Yiftach

File: content/rs/administering/active-active.md [hunk 3]
```diff
@@ -1,19 +1,17 @@
...
 alwaysopen: false
 categories: ["RS"]
+aliases: /rs/administering/intercluster-replication/crdbs/
 ---
-Conflict Free Replicated Databases (CRDBs) provide a simple and
-effective way to replicate your data between one or more Redis
-Enterprise Software (RS) clusters.
+In Redis Enterprise, active-active geo-distribution is based on CRDT technology.
+The Redis Enterprise implementation of CRDT is called Conflict-Free Replicated Database (CRDB).
+With CRDBs, applications can read and write to the same data set from different geographical locations
```
[the remark this answers is not in the document]
Comment (bmansheim) [reply content/rs/administering/active-active.md]: Didn't Yiftach want to specifically emphasize the read/write as opposed to other use cases.

File: content/rs/administering/active-passive.md [hunk 4]
```diff
@@ -1,29 +1,23 @@
 ---
-Title: Unidirectional Replication with Replica of
+Title: Active-Passive Geo-Distributed Redis (Replica-Of)
```
[the remark this answers is not in the document]
Comment (bmansheim) [reply content/rs/administering/active-passive.md]: yes

File: content/rs/administering/active-active.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bmansheim) [reply content/rs/administering/active-active.md]: He said "geo-distributed"

[the remark this answers is not in the document]
Comment (Paz-Yanover) [reply content/rs/administering/active-active.md]: Ok, we have few places in the documentation which say "geo-replicated", so maybe better to make is the same all over the place

**Pull request merged**